grakn-core/src/main/java/ai/grakn/util/Schema.java

```diff
@@ -116,6 +113,18 @@ public static boolean isMetaLabel(Label label) {
             }
             return false;
         }
+
+        @Nullable
+        @CheckReturnValue
+        public static MetaSchema valueOf(Label label){
+            if(THING.getLabel().equals(label)) return THING;
+            if(ENTITY.getLabel().equals(label)) return ENTITY;
+            if(ROLE.getLabel().equals(label)) return ROLE;
+            if(ATTRIBUTE.getLabel().equals(label)) return ATTRIBUTE;
+            if(RELATIONSHIP.getLabel().equals(label)) return RELATIONSHIP;
+            if(RULE.getLabel().equals(label)) return RULE;
+            return null;
+        }
```

Comment:
How about using something like `isMetaLabel` above?

```
public static MetaSchema valueOf(Label label) {
    for (MetaSchema metaSchema : MetaSchema.values()) {
        if (metaSchema.getLabel().equals(label)) return metaSchema;
    }
    return null;
}
```
and bonus:
```
public static boolean isMetaLabel(Label label) {
    return valueOf(label) != null;
}
```

Reply:
Good point. Not sure why I missed that